foundation/aissemble-foundation-core-python/src/policy_manager/policy/json/policy_input.py

```diff
@@ -67,3 +70,23 @@ class PolicyInput(BaseModel):
     The rules for this policy.
     """
     rules: List[PolicyRuleInput] = []
+
+    """
+    This attribute is deprecated and should not be used. Target are now represented as
+    a List of Target's instead of a single Target attribute 'target'.
+    This attribute is replaced by `targets`.
+    """
+    target: Optional[Target]
+
+    def getAnyTargets(self) -> List[Target]:
```

Comment:
This method is only used by the `abstract_policy_manager` for getting the `target` or `targets` from the JSON input. So the idea was it would check both attrubutes to support anyone still using the outdated `target` in their JSON policy. In the `Policy.py` it does property interception as that what the downstream user would be interacting with.

Reply:
This is what I mean: https://stackoverflow.com/a/17330273

Reply:
Yeah I totally missed that this was the JSON mapping class.  You can ignore this.